app/controllers/events/view/edit/attendee.js

```diff
@@ -0,0 +1,35 @@
+import Controller from '@ember/controller';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+
+export default Controller.extend(EventWizardMixin, {
+  async saveForms(data) {
+    for (const customForm of data.customForms ? data.customForms.toArray() : []) {
+      await customForm.save();
+    }
+    return data;
+  },
+  actions: {
+    async save(data) {
+      try {
+        await this.saveForms(data);
+      } catch (error) {
+        this.notify.error(this.ln10.t(error.message));
+      }
+      this.saveEventDataAndRedirectTo(
+        'events.view.index',
+        []
+      );
+    },
+    async move(direction, data) {
+      try {
+        await this.saveForms(data);
+      } catch (error) {
+        this.notify.error(this.ln10.t(error.message));
+      }
+      this.saveEventDataAndRedirectTo(
```

Comment:
Do not do this if there was an error.

Reply:
@niranjan94 What should be done in case of error? :sweat_smile:

Reply:
@uds5501 you can shift function on line 29 to try block.

Reply:
@shreyanshdwivedi OH! Okay, modifying the function!

Reply:
@shreyanshdwivedi @niranjan94 review now please